pages/common/git-scp.md

```diff
@@ -0,0 +1,26 @@
+# git scp
+
+> Copy files from the current working tree to the working directory of a remote repository.
+> Internally this script uses rsync and not scp as the name suggests.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-scp>.
+
+- Copy unstaged files to remote:
+
+`git scp staging`
+
+- Copy staged and unstaged files to remote:
+
+`git scp staging HEAD`
+
+- Copy files that has been changed in the last commit, plus any staged or unstaged files to remote:
+
+`git scp staging HEAD~1`
+
+- Copy specific files:
+
+`git scp staging {{path/to/file1}}...{{path/to/filen}}`
```

Comment:
```suggestion
`git scp staging {{path/to/file1 file2 ... filen}}`
```
I guess this can be used here

Reply:
This can be n no. of files. Don't you think the previous syntax give more clarity.

Reply:
> There can be n no. of files. Also the path can different for different files. 
Don't you think the previous syntax give more clarity?

Reply:
I am not sure, We generally use these multiple command arguments but we have been using it for files and directories too recently, It isn't documented in the style guide. So I will ask other's opinions.

cc @navarroaxel @EmilySeville7cfg

Reply:
If `...` is a mandatory syntax part of the command then we can leave the current code example as is without modification.

Reply:
> If `...` is a mandatory syntax part of the command then we can leave the current code example as is without modification. But at the end of the day @pg99696 should decide what is better based on our opinions. ;)

It's not mandatory, there can be 1 to N no. of files in the command. Please check this doc page for your reference. 
https://github.com/tj/git-extras/blob/master/Commands.md#git-scp